### PR TITLE
10.2 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/10/10.2.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* We fixed an issue where it an "oops" pop-up window was shown when an XML schema file was located in a drive that did not exist. (Ticket 191015)
+* We fixed an issue where a [Data Grid 2](/appstore/modules/data-grid-2/) module configured with an XPath data source with a `not equal` constraint did not apply the filters on associations that had been specified in the drop-down filter widget. (Ticket 191292)
+* We fixed an issue where re-importing a module with a nanoflow that called a microflow caused running native apps to be unable to call that microflow. Mendix Runtime operations for microflow calls now always have the same Mendix Runtime operation ID. (Ticket 191302)
+* We fixed an issue where an app containing an input widget with a page title override on one of its actions triggered an error and could not be opened. (Ticket 192138)
+* We fixed an issue where deployment failed when a Data Grid 2 module or a gallery widget used a database over an association with selection enabled. (Tickets 192500, 192875, 193161)
+* We fixed an issue where a button for creating a grid using a page parameter caused a deployment error. (Ticket 192593)
```
Comment (ancajuravle) [inline content/en/docs/releasenotes/studio-pro/10/10.2.md]: Adam, changing this to a "button for creating a grid" means totally something else than a (data) grid create button: https://docs.mendix.com/refguide/control-bar/#create-button. It's about the button within the (data) grid which is a create button, and by default is the New button.

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/10/10.2.md]: Thanks for bringing this up @ancajuravle - does this note work?:

`We fixed an issue where clicking the **New** grid button when using a page parameter caused a deployment error. (Ticket 192593)`

Comment (ancajuravle) [reply content/en/docs/releasenotes/studio-pro/10/10.2.md]: Yes, better this way.

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/10/10.2.md]: Thanks, fixed: https://github.com/mendix/docs/pull/6433/commits/1af876c55b3018cb61fcd062393b63bdd7b48a08

File: content/en/docs/releasenotes/studio-pro/10/10.2.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* We added support for the **Value** property of attribute types along with controls that depend on its value in the **Properties** pane while we hide the **Value** radio button for AutoNumber, Binary, and Hashed String attribute types in both the **Properties** pane and dialog boxes, because the value is always **Stored**.
+* We added the possibility to quickly filter properties in the **Properties** pane by the name of a property or a group.
+* A **Context** data source of a data view can now select an entity path from any surrounding data widget, including other data views.
+* We added the option to wait for all participant responses before completing a multi-user task in a workflow next to completing the task as soon as the final outcome is known. For more information, see the [Completion Moment](/refguide/multi-user-task/#completion-moment) section in *Multi-User Task*.
+* We added support for running Mendix apps on nested URL paths (for example, ` https://example.com/nested/path/mendix-app`).
+* We optimized the refreshing of the MxAssist Logic Bot menu, which can help with larger apps.
+* We improved the microflow and nanoflow return type dialog box usability.
```
Comment (exec-astraea) [inline content/en/docs/releasenotes/studio-pro/10/10.2.md]: We've got feedback about this feature from MVPs and it's better to rephrase this to:
"The return type of Microflows and Nanoflows is now determined automatically based on the expression used in the end event"

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/10/10.2.md]: Thanks - revised here: https://github.com/mendix/docs/pull/6433/commits/2384bd7e201cb9de2b7a9a8e46b5eafff529fe51
